docs/env-vars.md

```diff
@@ -12,12 +12,13 @@ Typeagent currently supports two families of environment variables:
 ## OPENAI environment variables
 
 The (public) OpenAI environment variables include:
-
```

Comment:
Can I get this blank line back? I like the extra bit of whitespace in the rendered page.